translations/en.json

```diff
@@ -1,39 +1,40 @@
 {
-  "dispensing": "Dispensing",
+  "active": "Active",
```

Comment:
I alphabetized these because it was annoying my sense of order... :)

Reply:
I might need to add some config settings to automatically organize this going forward and remove any unused translations.

Reply:
@pirupius  that would be great!